source/wireversion-featurelist.md

```diff
@@ -0,0 +1,33 @@
+| 5.3            | 16           |                                                                                                                                                                                                                                                                             |
+| 6.0            | 17           | <p>Support for Partial Indexes</p><p>Sharded Time Series Collections</p><p>FCV set to 5.0</p>                                                                                                                                                                               |
+| 6.1            | 18           | <p>Update Perl Compatible Regular Expressions version to PCRE2</p><p>Add `*UCP` option for regex queries</p>                                                                                                                                                                |
+| 6.2            | 19           | <p>Collection validation ensures BSON documents conform to BSON spec</p><p>Collection validation checks time series collections for internal consistency</p>                                                                                                                |
+| 7.0            | 21           | <p>Atlas Search Index Management</p><p>`$currentOp` aggregation Metrics</p><p>Compound Wildcard Indexes</p><p>Support large change stream events via `$changeStreamSplitLargeEvent` stage</p><p>`serverStatus` output gets new fields</p> <p>Slot Based Query Execution</p> |
+| 7.1            | 22           | <p>Improved Index Builds</p><p>Exhaust Cursors Enabled for Sharded Clusters</p><p>New Sharding Statistics for Chunk Migrations</p><p>Self-Managed Backups of Sharded Clusters                                                                                               |
+| 7.2            | 23           | <p>Database Validation on `mongos` Aggregation Queries</p><p>`serverStatus` Metrics</p><p>Default Chunks Per Shard</p>                                                                                                                                                      |
+| 7.3            | 24           | <p>Compaction Improvements</p><p>New `serverStatus` metrics</p>                                                                                                                                                                                                             |
+| 8.0            | 25           |                                                                                                                                                                                                                                                                             |
```

Comment:
What about 8.0? Range Encryption GA, new bulkWrite command, OIDC authentication, read concern "snapshot" on capped collections: https://www.mongodb.com/docs/upcoming/release-notes/8.0/#std-label-release-notes-8.0

Reply:
Thanks! Don't know how I missed that one.